apps/coinsence-kit/contracts/CoinsenceKit.sol

```diff
@@ -73,8 +73,8 @@ contract CoinsenceKit is KitBase {
         Coin coin;
 
         bytes32[2] memory appIds = [
-            apmNamehashOpen("coinsence-space"),     // 0
-            apmNamehashOpen("coinsence-coin")       // 1
+            apmNamehash("coinsence-space"),     // 0
+            apmNamehash("coinsence-coin")       // 1
```

Comment:
attention, attention! :) 
we should really fix this as the next thing...

Reply:
yes we should :\ I forget to change it every time I deploy the kit locally XD